### ISSUE TITLE
Apply Googles HTTPs policy

### ISSUE THREAD

**Issue by ellcs** [rogerwelin/sinatra-chartkick-example#1]
Since google does supply `https` only, the url within chart.erb should be changed:

from:
```html
<script src="//www.google.com/jsapi"></script>
```
to, by adding `https:`:
```html
<script src="https://www.google.com/jsapi"></script>
```

**Comment by ellcs** [comment]
Anyway thank you for providing this example. :)

**Comment by rogerwelin** [comment]
@ellcs Thx for informing, I will fix

**Comment by rogerwelin** [comment]
fixed in [0ad95ae](https://github.com/rogerwelin/sinatra-chartkick-example/commit/0ad95aeddd5147a7917f41e1ed79b3409afd7021)
